### ISSUE TITLE
Copy pasting text in editor keeps formatting from source

### ISSUE THREAD

**Issue by baudren** [baudren/NoteOrganiser#85]
Right now, copy pasting text directly in the editors does not use the font of the editor. The text is well understood, and rendered in markdown, but it would be better to fix this.

**Comment by egolus** [comment]
yes, I noticed this, too. It copies the formatting from the source. In many programs there's an option to just paste the text itself without any formatting. It looks like we have to build this for our editors
